Project.toml

```diff
@@ -1,23 +1,26 @@
 name = "ClassImbalance"
 uuid = "04a18a73-7590-580c-b363-eeca0919eb2a"
 authors = ["Paul Stey <paulstey@gmail.com>", "Dilum Aluthge <dilum@aluthge.com>", "Brown Center for Biomedical Informatics <bcbi@brown.edu>"]
-version = "0.8.7"
+version = "0.8.8"
```

Comment:
I think this should be a minor release (`"0.9.0"` or `"0.9.0-dev"`)

Reply:
alright, I'll do that